File: app/src/main/java/sergey/yatsutko/siberiancoal/data/entity/geocoderPojo/Thoroughfare__.java

package sergey.yatsutko.siberiancoal.data.entity.geocoderPojo;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Thoroughfare__ {

    @SerializedName("ThoroughfareName")
    @Expose
    private String thoroughfareName;

    public String getThoroughfareName() {
        return thoroughfareName;
    }

    public void setThoroughfareName(String thoroughfareName) {
        this.thoroughfareName = thoroughfareName;
    }

}

File: app/src/main/java/sergey/yatsutko/siberiancoal/data/entity/geocoderPojo/Thoroughfare.kt
package sergey.yatsutko.siberiancoal.data.entity.geocoderPojo

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Thoroughfare {

    @SerializedName("ThoroughfareName")
    @Expose
    lateinit var thoroughfareName: String
    @SerializedName("Premise")
    @Expose
    lateinit var premise: Premise

}
